9.4/Dockerfile.rhel7

```diff
@@ -70,6 +72,11 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
 
 VOLUME ["/var/lib/pgsql/data"]
 
+# {APP_DATA} needs to be accessed by postgres user while s2i assembling
+# postgres user changes permissiond of files in APP_DATA during assemblng
+RUN chown -R 26:0 ${APP_DATA} && \
+    useradd -G root 26
```

Comment:
This doesn't work on my system.  You probably want `usermod -a -G root postgres`.
I don't think removal of `chmod` was correct, because default umask should be 0022 -> which means that group get's only `r-x` permissions.